### DISCUSSION TITLE
Permission to make a port for Warp terminal

### DISCUSSION THREAD

**Discussion by chris-c-thomas** [daltonmenezes/aura-theme#120] [Port request]
I'd like to port Aura theme to the newish Warp terminal. It can be found at https://www.warp.dev

Should be straight forward. You do need to request access to the app since it's still in beta but it's a really great terminal that I'm enjoying so far (my main terminal is Hyper).

@daltonmenezes if it's okay with you I'd like to go forward with it. 

Thanks.

**Comment by chris-c-thomas** [comment]
I should point out that after requesting access, you can join their Discord, and I think that instantly gives you access. At least, it did for me.

**Comment by daltonmenezes** [comment]
@chris-c-thomas it would be awesome if you work on it. Just let me know if you need some help.

Just to make this process easier as possible, take a look at the [docs](https://github.com/daltonmenezes/aura-theme#documentation), but I'm here if you need.  💜

**Comment by daltonmenezes** [comment]
Hi @chris-c-thomas , are you still working on this port?

**Comment by andypiper** [comment]
Just started playing with Warp - is anyone actively working on an Aura theme?

**Reply by daltonmenezes** [reply]
@andypiper I asked to @chris-c-thomas about the progress, but he didn't respond to me. If you are interested in creating the port just let me know, so I can assign you to a related issue. 💜

**Reply by chris-c-thomas** [reply]
Apologies. I never was able to get around to it. Feel free to go ahead! Warp is pretty great!

**Reply by dklaver15** [reply]
@daltonmenezes I could probably work on this one too.  I’ve made a lot of progress on the port for Nova and if I remember correctly, themes for Warp are pretty easy. But if someone else would rather take it that’s fine.

**Reply by andypiper** [reply]
I do not have the bandwidth to do it right now, but wanted to see if someone had one stashed somewhere :-) thanks all.

**Reply by daltonmenezes** [reply]
@chris-c-thomas @andypiper no problem, @DanKlaver15 when you have finish the Nova port, just let me know if you yet have interest to make the Wrap port, please, so I can make an issue for this and assign you, Thanks 💜

**Comment by alexmck** [comment]
Hi @daltonmenezes. I have an Aura port for Warp ready to go. Would you like me to submit a pull request?

**Reply by daltonmenezes** [reply]
@alexmck sure, it would be great!
